src/components/dropdownSearch/MultiSelectDropdownSearch/tests/MultiSelectDropdownSearchConnected.spec.tsx

```diff
@@ -0,0 +1,167 @@
+import {UUID} from '../../../../utils/UUID';
+import * as React from 'react';
+import {mount, ReactWrapper} from 'enzyme';
+import {IDropdownSearchProps} from '../../DropdownSearch';
+import {Provider, Store} from 'react-redux';
+import {IReactVaporState} from '../../../../ReactVapor';
+import {MultiSelectDropdownSearchConnected} from '../MultiSelectDropdownSearchConnected';
+import {MultiSelectDropdownSearch} from '../MultiSelectDropdownSearch';
+import {TestUtils} from '../../../../utils/TestUtils';
+import {clearState} from '../../../../utils/ReduxUtils';
+import {defaultSelectedOption} from '../../DropdownSearchReducers';
+import {
+  closeDropdownSearch,
+  openDropdownSearch,
+  updateOptionsDropdownSearch
+} from '../../DropdownSearchActions';
+
+describe('MultiSelectDropdownSearch', () => {
+  const id: string = UUID.generate();
+
+  describe('<MultiSelectDropdownSearchConnected />', () => {
+    let wrapper: ReactWrapper<any, any>;
+    let multiSelectDropdownSearchConnected: ReactWrapper<IDropdownSearchProps, any>;
+    let store: Store<IReactVaporState>;
+
+    const defaultOptions = [{ value: 'a', displayValue: 'a' }, { value: 'b', displayValue: 'b' }];
+
+    const renderMultiSelectDropdownSearchConnected = () => {
+      wrapper = mount(
+        <Provider store={store}>
+          <MultiSelectDropdownSearchConnected id={id} defaultOptions={defaultOptions} />
+        </Provider>,
+        { attachTo: document.getElementById('App') },
+      );
+      multiSelectDropdownSearchConnected = wrapper.find(MultiSelectDropdownSearch).first();
+    };
+
+    beforeEach(() => {
+      store = TestUtils.buildStore();
+    });
+
+    afterEach(() => {
+      store.dispatch(clearState());
+      wrapper.unmount();
+      wrapper.detach();
+    });
+
+    describe('mount and unmount', () => {
+
+      beforeEach(() => {
+        renderMultiSelectDropdownSearchConnected();
+      });
+
+      it('should call onMount prop when mounted', () => {
+        wrapper.unmount();
+        store.dispatch(clearState());
+        expect(store.getState().dropdownSearch.length).toBe(0);
+
+        wrapper.mount();
+        expect(store.getState().dropdownSearch.length).toBe(1);
+      });
+
+      it('should call onDestroy prop when will unmount', () => {
+        wrapper.unmount();
+        expect(store.getState().dropdownSearch.length).toBe(0);
+      });
+    });
+
+    describe('mapDispatchToProps', () => {
+
+      beforeEach(() => {
+        renderMultiSelectDropdownSearchConnected();
+      });
+
```

Comment:
that is to be consistent with how @pochretien implemented his Connected Components tests. It verifies if the component has all the method it needs when rendered.